perfeed/tools/pr_summarizer.py

```diff
@@ -8,16 +8,19 @@
 from perfeed.git_providers.base import BaseGitProvider
 from perfeed.llms.base_client import BaseClient
 from perfeed.llms.ollama_client import OllamaClient
-from perfeed.models.pr_summary import PRSummary
+from perfeed.models.pr_summary import PRSummary, PRSummaryMetadata
 from perfeed.utils import json_output_curator
+from datetime import datetime, timezone
+from typing import Tuple
+
 
 class PRSummarizer:
     def __init__(self, git: BaseGitProvider, llm: BaseClient):
         self.git = git
         self.llm = llm
 
-    async def run(self, repo: str, pr_number: int) -> PRSummary:
```

Comment:
Please keep `async` since we need to run it in parallel.

You can use `pr_summary = asyncio.run(summarizer.run("perfeed", 5))` to invoke an async function in a non-async func.

Reply:
can u go test the pr_summarizer.py? did not work for me, therefore the change
`<coroutine object PRSummarizer.run at 0x7f6ada10be00>
sys:1: RuntimeWarning: coroutine 'PRSummarizer.run' was never awaited`

Reply:
Take a look at https://github.com/Perfeed/perfeed/pull/16 and approve. It should show you how to run it. This function has to be `async` in order to scale.